backend/app/scripts/create_post.py

```
@@ -0,0 +1,377 @@
+import traceback
+from typing import List, Optional
+import firebase_admin.firestore
+import requests
+import random
```

Ruff (0.8.2)

`random` imported but unused

Remove unused import: `random`

(F401)